legal-api/tests/unit/services/test_authorization.py

```diff
@@ -25,9 +25,9 @@
 
 import pytest
 from flask import jsonify
-from registry_schemas.example_data import ALTERATION_FILING_TEMPLATE, ANNUAL_REPORT, CORRECTION_AR, \
-    CHANGE_OF_REGISTRATION_TEMPLATE, RESTORATION, FILING_TEMPLATE, DISSOLUTION, PUT_BACK_ON, CONTINUATION_IN, \
-    CONSENT_CONTINUATION_OUT, CONTINUATION_OUT
+from registry_schemas.example_data import AGM_LOCATION_CHANGE, ALTERATION_FILING_TEMPLATE, ANNUAL_REPORT, \
+    CORRECTION_AR, CHANGE_OF_REGISTRATION_TEMPLATE, RESTORATION, FILING_TEMPLATE, DISSOLUTION, PUT_BACK_ON, \
+    CONTINUATION_IN, CONSENT_CONTINUATION_OUT, CONTINUATION_OUT
 
```

Comment:
Currently waiting on bcgov/entity#17941 to get merged to use the `AGM_LOCATION_CHANGE` example data.